app/assets/stylesheets/article-show.scss

```diff
@@ -779,24 +763,9 @@ article {
         font-size: 0.85em;
 
         a {
-          @include themeable(color, theme-secondary-color, $medium-gray);
+          color: var(--container-color-tertiary);
           margin-right: 6px;
           display: inline-block;
-
-          .icon-img {
```

Comment:
I basically replaced `<img>` in couple views with inline SVGs so it's easier to style them depending on theme.